source/getting-started/supported-databases.txt

```diff
@@ -38,28 +38,34 @@ Relational Migrator supports the following source databases:
      - Supported Versions 
      - Deployments 
 
-   * - Oracle 
+   * - :ref:`Oracle <rm-prereq-oracle>`
      - 11g and higher
      - Self hosted, AWS RDS
 
-   * - Microsoft SQL Server
+   * - :ref:`Microsoft SQL Server <rm-prereq-sqlserver>`
      - 2012 and higher (Enterprise or Developer edition required for
        versions before 2016)
      - Self hosted, AWS RDS, Azure SQL Database
 
-   * - PostgreSQL [1]_
+   * - :ref:`PostgreSQL <rm-prereq-postgres>`
      - 10 and higher
      - Self hosted, AWS RDS/Aurora, Azure Database
 
-   * - MySQL
+   * - `TimescaleDB hypertables <https://docs.timescale.com/use-timescale/latest/hypertables/>`__
```

Comment:
Nitpick but TimescaleDB is an extension of Postgres and not really a standalone database, perhaps it can be a subheader or sublink within Postgres?

Reply:
Oh I just noticed that the Jira ticket calls it out - it's OK to leave it on its own but it's a much smaller market-share and we can count on one hand the number of times this has been attempted since support for Timescale was launched. Postgres is probably the # 1 migration source at the moment conversely.